app/src/features/apiClient/hooks/useApiClientFileImporter.ts

```diff
@@ -190,37 +196,38 @@ const useApiClientFileImporter = (importer: ImporterTypes) => {
       batchWrite(BATCH_SIZE, apis, handleApiWrites),
     ]);
 
-    if (failedCollectionsCount > 0) {
-      const failureMessage =
-        failedCollectionsCount > 0
-          ? `${failedCollectionsCount} ${failedCollectionsCount > 1 ? "collections" : "collection"} failed`
-          : "";
-      if (failureMessage.length) {
-        toast.warn(`Some imports failed: ${failureMessage}, Please contact support if the issue persists.`);
-      }
+    if (failedCollectionsCount > 0 || failedApisCount > 0) {
+      toast.warn(
+        `Failed to import ${
+          failedCollectionsCount + failedApisCount
+        } items. Please contact support if the issue persists.`
+      );
     }
 
-    return importedCollectionsCount;
+    return { importedCollectionsCount, importedApisCount };
   }, [user, workspace, onSaveRecord, collections, apis]);
 
   const handleImportData = useCallback(
     async (onSuccess: () => void) => {
       setIsImporting(true);
-      trackImportApiCollectionsStarted(recordsCount, environments.length);
 
       try {
         const [envResult, collResult] = await Promise.allSettled([
           handleImportEnvironments(),
           handleImportCollectionsAndApis(),
         ]);
         const importedEnvironments = envResult.status === "fulfilled" ? envResult.value : 0;
-        const importedCollectionsAndApis = collResult.status === "fulfilled" ? collResult.value : 0;
+        const importedCollections = collResult.status === "fulfilled" ? collResult.value.importedCollectionsCount : 0;
+        const importedApis = collResult.status === "fulfilled" ? collResult.value.importedApisCount : 0;
+        const importedCollectionsAndApis = importedCollections + importedApis;
+
+        trackImportSuccess("requestly", importedCollections, importedApis);
 
         const failedEnvironments = environments.length - importedEnvironments;
         const failedCollectionsAndApis =
           (collections.length ? collections.length : apis.length) - importedCollectionsAndApis;
 
-        if (!importedEnvironments && !importedCollectionsAndApis) {
+        if (!importedEnvironments && !importedCollections && !importedApis) {
```

Comment:
What if the files being imported did not have any collections or APIs? Then this toast is incorrect. Right? Is that case handled?